WalletWasabi.Gui/Behaviors/FocusOnAttachedBehavior.cs

```diff
@@ -0,0 +1,19 @@
+using Avalonia.Controls;
+using Avalonia.Threading;
+using Avalonia.Xaml.Interactivity;
+
+namespace WalletWasabi.Gui.Behaviors
+{
+	public class FocusOnAttachedBehavior : Behavior<Control>
+	{
+		protected override void OnAttached()
+		{
+			base.OnAttached();
+
+			Dispatcher.UIThread.Post(() =>
+			{
+				AssociatedObject.Focus();
+			}, DispatcherPriority.Layout);
```

Comment:
Is this the solution to trigger something when control is visible for the user?

Reply:
Yes, people sometime use Delays, this is telling Avalonia UIThread dispatcher, queue this after layout has happened.

Reply:
its a solution for inside control code, not viewmodel code.